### PR TITLE
Adding security and contribution guidelines

### PR DESCRIPTION
**Pull request by chayim**
The guidelines in this pull request come from their redis equivalents. The original [security policy](https://github.com/redis/redis/blob/unstable/SECURITY.md) and [contribution policy](https://github.com/redis/redis/blob/unstable/CONTRIBUTING.md) policies have had their text modified in this pull request, to make them redis-stack specific.

### CODE REVIEW

**Review by gkorland — APPROVED** [review]
I also just added our CLA

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+Note: by contributing code to this project in any form, including sending
+a pull request via Github, a code fragment or patch via private email or
```
Comment (yossigo) [inline CONTRIBUTING.md:2]:
```suggestion
a pull request via GitHub, a code fragment or patch via private email or
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+public discussion groups, you agree to release your code under the terms
+of the [LICENSEs](license.txt).
+
+# IMPORTANT: HOW TO USE REDIS GITHUB ISSUES
+
+Github issues SHOULD ONLY BE USED to report bugs, and for DETAILED feature
```
Comment (yossigo) [inline CONTRIBUTING.md:8]:
```suggestion
GitHub issues SHOULD ONLY BE USED to report bugs, and for DETAILED feature
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    c. Push to your branch (git push origin my_branch)
+    d. Initiate a [pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request) on this repository.
+    e. Done :)
+
+3. Keep in mind that we are very overloaded, so issues and PRs sometimes wait
+for a *very* long time. However this is not lack of interest, as the project
+gets more and more users, we find ourselves in a constant need to prioritize
+certain issues/PRs over others. If you think your issue/PR is very important
+try to popularize it, have other users commenting and sharing their point of
+view and so forth. This helps.
```
Comment (yossigo) [inline CONTRIBUTING.md:36]: Do we stand behind this? Seems like antirez wrote this a long time ago and we never refreshed it for Redis. Maybe it's time to do a complete overhaul of this file and then sync back to Redis.

File: SECURITY.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 7.0.x   | :white_check_mark: |
+| 6.2.x   | :white_check_mark: |
+
+## Reporting a Vulnerability
+
+If you believe you’ve discovered a serious vulnerability, please contact the
+Redis core team at redis@redis.io. We will evaluate your report and if
```
Comment (yossigo) [inline SECURITY.md:18]: I suggest to have a separate address for Redis Stack.

File: SECURITY.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+necessary issue a fix and an advisory. If the issue was previously undisclosed,
+we’ll also mention your name in the credits.
+
+## Responsible Disclosure
```
Comment (yossigo) [inline SECURITY.md:22]: Shoudl this apply to Redis Stack? Not sure.

File: CONTRIBUTING.md (hunk 3 above)
Comment (chayim) [reply CONTRIBUTING.md:36]: Fair question. We tend to be very responsive here in redis-stack, so I can vote for removing, we also don't have it in the modules. I'll remove.

File: SECURITY.md (hunk 5 above)
Comment (chayim) [reply SECURITY.md:22]: I generally think security researchers want this. I see this ask all time across other projects, and repositories. It seems consistent with common practice *and* we have the release drafter for this.

**Pull request merged**